drop/modules/aberrant-splicing-pipeline/FRASER/07_extract_results_FraseR.R

```diff
@@ -41,9 +41,7 @@ setAutoBPPARAM(MulticoreParam(snakemake@threads))
 
 # Load data and annotate ranges with gene names
 fds <- loadFraserDataSet(dir=workingDir, name=dataset)
-GRCh <- ifelse(snakemake@config$genomeAssembly == 'hg19', 37, 
-               ifelse(snakemake@config$genomeAssembly == 'hg38', 38,
-                      error('Genome assembly must be either hg19 or hg38')))
+GRCh <- ifelse(snakemake@config$genomeAssembly %in% c('hg19', 'hs37d5'), 37, 38)
```

Comment:
This check could be done in `DropConfig` and `GRCh` retrieved in the snakemake header